app/services/youtube_auth.py

```
@@ -0,0 +1,89 @@
+        RuntimeError: If no valid credentials are available.
+    """
+    token_file = settings.YOUTUBE_OAUTH_TOKEN_FILE
+    client_secrets_file = settings.YOUTUBE_CLIENT_SECRETS_FILE
+    creds = None
+
+    # Load existing token
+    if os.path.exists(token_file):
+        try:
+            creds = Credentials.from_authorized_user_file(token_file, SCOPES)
+            logger.debug("YouTube OAuth token loaded from file.")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)